src/models/globalAccessControl.ts

```diff
@@ -146,10 +141,10 @@ export class GlobalAccessControl extends Model {
     return this._globalAccessControlData.analyticRights
   }
 
-  eagerUserProfile () {
+  eagerUser () {
     if (typeof this._globalAccessControlData.user === 'undefined') {
       return undefined
```

Comment:
Don't use the `undefined` global.  Make this a one line ternary like the other `eager` methods.